components/Buttons/src/MDCButton.m

```diff
@@ -686,13 +686,10 @@ - (void)setBackgroundColor:(UIColor *)backgroundColor forState:(UIControlState)s
 
 - (CGFloat)elevationForState:(UIControlState)state {
   NSNumber *elevation = _userElevations[@(state)];
-  if (state != UIControlStateNormal && !elevation) {
+  if (state != UIControlStateNormal && elevation == nil) {
     elevation = _userElevations[@(UIControlStateNormal)];
   }
-  if (elevation) {
-    return (CGFloat)[elevation doubleValue];
-  }
-  return 0;
+  return (CGFloat)elevation.doubleValue;
```

Comment:
FYI, we usually prefer an explicit value instead of relying on the Objective-C runtime to return zero when the pointer is nil. The reason is twofold: it shows the programmer thought about the case and deliberately chose zero, and if the value to return changes it can be easily updated.

I usually prefer the `?:` form below, but that's up to the individual programmer.

Reply:
Oh, I did not know that. Thanks.

I used a ternary instead since we want to unbox the value.